aws/resource_aws_batch_job_queue_test.go

```diff
@@ -94,6 +100,11 @@ func TestAccAWSBatchJobQueue_disappears(t *testing.T) {
 				),
 				ExpectNonEmptyPlan: true,
 			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
```

Comment:
Since `_disappears` tests are removing the resource in question to verify Terraform triggers recreation of the resource instead of an error, these tests will generate the below error when trying to do `ImportState` testing:

```
--- FAIL: TestAccAWSBatchJobQueue_disappears (80.44s)
    testing.go:640: Step 1 error: Resource specified by ResourceName couldn't be found: aws_batch_job_queue.test_queue
```

The solution here is just to not include the import testing in "disappears" tests like these. 👍